examples/storm-kafka-examples/src/main/java/org/apache/storm/kafka/trident/KafkaProducerTopology.java

```diff
@@ -54,7 +54,7 @@ public static StormTopology newTopology(String brokerUrl, String topicName) {
     /**
      * @return the Storm config for the topology that publishes sentences to kafka using a kafka bolt.
      */
-    private static Properties newProps(final String brokerUrl, final String topicName) {
+    static Properties newProps(final String brokerUrl, final String topicName) {
```

Comment:
I use this for some test cases I have I use that are not part of the codebase. This is not a change as it's left as package protected